nucypher-core/src/message_kit.rs

```diff
@@ -17,7 +17,8 @@ pub struct MessageKit {
     /// Encapsulated symmetric key for this message.
     pub capsule: Capsule,
     #[serde(with = "serde_bytes")]
-    ciphertext: Box<[u8]>,
+    /// Ciphertext of the message.
+    pub ciphertext: Box<[u8]>,
```

Comment:
Why do you need to expose it?

Reply:
No reason, that was an accidental change. I'll revert it.